airbyte-integrations/connectors/destination-bigquery/src/integrationTestLegacy/kotlin/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGeneratorIntegrationTest.kt

```diff
@@ -1,620 +1,63 @@
 /*
- * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
  */
+
 package io.airbyte.integrations.destination.bigquery.typing_deduping
 
 import com.fasterxml.jackson.databind.JsonNode
 import com.fasterxml.jackson.databind.node.ObjectNode
-import com.google.cloud.bigquery.*
-import edu.umd.cs.findbugs.annotations.SuppressFBWarnings
-import io.airbyte.cdk.integrations.base.JavaBaseConstants
-import io.airbyte.commons.json.Jsons.deserialize
+import com.google.cloud.bigquery.FieldValueList
+import com.google.cloud.bigquery.Schema
+import com.google.cloud.bigquery.StandardSQLTypeName
+import com.google.cloud.bigquery.TableResult
 import io.airbyte.commons.json.Jsons.deserializeExact
 import io.airbyte.commons.json.Jsons.emptyObject
 import io.airbyte.commons.json.Jsons.jsonNode
-import io.airbyte.integrations.base.destination.typing_deduping.*
-import io.airbyte.integrations.destination.bigquery.BigQueryConsts
-import io.airbyte.integrations.destination.bigquery.BigQueryDestination.Companion.getBigQuery
-import io.airbyte.integrations.destination.bigquery.migrators.BigQueryDestinationState
-import java.nio.file.Files
-import java.nio.file.Path
-import java.time.Duration
-import java.util.*
-import java.util.Map
-import java.util.stream.Collectors
-import kotlin.collections.LinkedHashMap
-import kotlin.collections.List
-import kotlin.collections.emptyList
-import kotlin.collections.indices
-import org.apache.commons.text.StringSubstitutor
-import org.junit.jupiter.api.Assertions
-import org.junit.jupiter.api.BeforeAll
-import org.junit.jupiter.api.Disabled
-import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.parallel.Execution
-import org.junit.jupiter.api.parallel.ExecutionMode
-import org.junit.jupiter.params.ParameterizedTest
-import org.junit.jupiter.params.provider.ValueSource
-import org.slf4j.Logger
-import org.slf4j.LoggerFactory
-
```

Comment:
I could be convinced that we should keep this, but it's largely redundant with the bulk CDK's tests. There are a few interesting tests (e.g. testCreateTableInOtherRegion) but de facto I don't think we're going to touch that code anytime soon.